### PR TITLE
Basic Routing setup

### LINKED ISSUE #1: Setup Basic Routing in the app
- Create basic routing in the app
- Import the component library
- Create placeholder components for pages that are not yet completed.

**Acceptance criteria**: Header with links that can be used to navigate to other pages

### PR DESCRIPTION
**Pull request by ttahm3d**
PR for Issue #1 

- Setup routing for various pages 
- Created Header
localhost_3000_

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Header from "./Header/Header";
+
+export { Header };
```
Comment (kjais1720) [inline src/components/index.js:3]: you can directly export it like 
`export { Header } from "./Header/Header"` (if its a named export)

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,18 @@
+@import url("https://chunksui.netlify.app/css/style.css");
```
Comment (kjais1720) [inline src/index.css:1]: I think linking the css in `index.html` is better for performance since we can use features like `preconnect` for stylesheets

**Review by kjais1720 — APPROVED** [review]
Looks good overall just a couple of small changes suggested

File: src/components/index.js (hunk 1 above)
Comment (ttahm3d) [reply src/components/index.js:3]: I want to have a common point for all my common components to be exported.

**Review by nakulsharma15 — APPROVED** [review]
Everything Looks Good. I liked the way you named your variables and files

**Pull request merged**